arlo-client/src/components/Audit/RoundManagement/RoundProgress.tsx

```diff
@@ -1,8 +1,71 @@
 import React from 'react'
+import { ProgressBar } from '@blueprintjs/core'
+import styled from 'styled-components'
 import H2Title from '../../Atoms/H2Title'
+import { IRound } from '../useRoundsJurisdictionAdmin'
+import useAuditBoards from '../useAuditBoards'
 
-const RoundProgress = () => {
-  return <H2Title>Round 1 Progress</H2Title>
+const MainBarWrapper = styled.div`
+  margin-bottom: 20px;
+  width: 500px;
+  font-size: 16px;
+`
+
+const SmallBarWrapper = styled.div`
+  margin-bottom: 10px;
+  width: 400px;
+`
+
+const RoundProgress = ({
+  electionId,
+  jurisdictionId,
+  round,
+}: {
+  electionId: string
+  jurisdictionId: string
+  round: IRound
+}) => {
+  const [auditBoards] = useAuditBoards(electionId, jurisdictionId, [round])
+  if (!auditBoards) return null
+  const [auditedBallots, sampledBallots] = auditBoards.reduce(
+    (a, v) => {
+      a[0] += v.currentRoundStatus.numAuditedBallots // eslint-disable-line no-param-reassign
+      a[1] += v.currentRoundStatus.numSampledBallots // eslint-disable-line no-param-reassign
```

Comment:
I know it's all local code, but I'd still prefer `a.numAudited` over `a[0]`

Reply:
alternatively, could do something like:

```
const sum = (ns: number[]) => ns.reduce((a, b) => a+b)
const auditedBallots = sum(auditBoards.map(ab => ab.currentRoundStatus.numAuditedBallots))
const sampledBallots = sum(auditBoards.map(ab => ab.currentRoundStatus.numSampledBallots))
```

Reply:
That does seem more self-documenting, thanks!